crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -28,6 +28,30 @@ pub struct DiagnosticMeasurement {
     pub value: f64,
 }
 
+/// Enum for display percision. Choose `IntegerValue` for discrete values and `DecimalValue(num_of_decimals)` to specify the number of decimals
+#[derive(Debug)]
+pub enum DisplayPercision {
+    IntegerValue,
+    DecimalValue(usize),
+}
+
+impl DisplayPercision {
+    pub fn get_num_of_decimals(&self) -> usize {
+        match self {
+            DisplayPercision::IntegerValue => 0,
+            DisplayPercision::DecimalValue(x) => *x,
+        }
+    }
+}
+
+// trait DiagnosticDisplayPercision {
+//     type DisplayPercision;
+// }
```

Comment:
Please do not leave intentionally commented out code in the codebase. If it's not meant to be included, please remove it. Otherwise, please complete the implementation.

Reply:
I left it in in regards to this: 
> Also we put some trait code in comments that we tried to use so we don't have to do any new imports but it didn't work. Could it be done using this? Or is there maybe an other way?

But if there is no answer I'll remove it :)